src/Nav/index.js

```diff
@@ -12,32 +12,45 @@ governing permissions and limitations under the License.
 import React from 'react'
 import PropTypes from 'prop-types'
 import { withPrefix } from 'gatsby'
-import { defaultFocus } from '../ManifestUtils'
 import { isExternal } from '../utils'
 
 import '@spectrum-css/sidenav'
 import './index.css'
 
-const nav = (data, defaultFocus, currDepth, maxDepth) => {
+const isPathSelected = (path, selected) => {
+  const compare = !selected.endsWith('/')
```

Comment:
I think this would read better as:

```
selected.endsWith('/')
?  selected.substring(0, selected.length - 1)
: selected
```

WDYT?

Reply:
I think it can even be simplied as:

`selected.replace(/\/$/, '');`

but it'd need to be tested

Reply:
agreed with the first ternary, it's a bit clearer.

the `replace` approach isn't as clear to me at first glance though.

Reply:
lol, I debated longer than I care to admit before commiting that ternary. I'll switch it. I'll skip the regex for now.